TwitchPlaysAssembly/Src/ComponentSolvers/ComponentSolver.cs

```diff
@@ -961,6 +962,62 @@ private void AwardSolve(string userNickName, int ComponentValue)
 		}
 	}
 
+	private void AwardRewardStrike(int strikeCount)
```

Comment:
We shouldn't be duplicating so much code. Could you make `userNickName` an optional argument of AwardStrike instead?